src/NetworkEntities.js

```diff
@@ -122,8 +132,11 @@ class NetworkEntities {
   }
 
   addEntityToPage(entity, parentId) {
+    var nonNetworkedParent = document.querySelector(`#${parentId}`)
     if (this.hasEntity(parentId)) {
       this.addEntityToParent(entity, parentId);
+    } else if (nonNetworkedParent) {
```

Comment:
I would recommend setting line 135 to just `var nonNetworkedParent;` and then the assignment for nonNetworkedParent could instead happen on this line as:

`else if ((nonNetworkedParent = document.querySelector(`#${parentId}`)))`

That way we can avoid running the querySelector call if we enter the first if condition.